image-build-push.sh

```diff
@@ -0,0 +1,36 @@
+# Terminal output colors 
+GREEN='\033[0;32m'
+ENDCOLOR='\033[0m' # No Color
+RED='\033[0;31m'
+
+# Environment Variables 
+PREFER_PODMAN="${PREFER_PODMAN:-0}"
+PUSH="${PUSH:-1}"
+TAG="${TAG:-test}"
```

Comment:
```suggestion
TAG="${TAG:-v0.1.0}"
```

Reply:
Because this file will be for development I've change the tag to 'dev'.